explore/launch/explore.launch

```diff
@@ -1,15 +1,16 @@
 <launch>
 <node pkg="explore_lite" type="explore" respawn="false" name="explore" output="screen">
-  <param name="robot_base_frame" value="base_link"/>
+  <param name="robot_base_frame" value="create1_tf/base_footprint"/>
```

Comment:
BTW, this is a robot-agnostic package so you shouldn't make this change here.

In fact, you should make these changes in the `create_autonomy` repo. And the one that I mentioned about the YAML too.

Reply:
I removed the mentions to the robot in the code, and then the needed changes will be applied from a YAML in create_autonomy.